src/handlers/payout/action.ts

```diff
@@ -422,9 +422,12 @@ export const handleIssueClosed = async (
     assigneeComment =
       `#### ${title} Reward \n### [ **[ CLAIM ${priceInEth} ${tokenSymbol.toUpperCase()} ]** ](${payoutUrl})\n` + "```" + shortenRecipient + "```";
     const permitComments = incentivesCalculation.comments.filter((content) => {
-      const permitUrlMatches = content.body.match(incentivesCalculation.claimUrlRegex);
-      if (!permitUrlMatches || permitUrlMatches.length < 2) return false;
-      else return true;
+      if (content.body.includes("https://pay.ubq.fi") && content.user.type == UserType.Bot) {
+        const url = new URL(content.body);
```

Comment:
As far as I know URL() accepts only a URL not a whole string with a bunch of other things

Reply:
sorry, I was just retrieving the old code. I should've marked it as draft. I'll mark it as draft for now and request review when it's ready

Reply:
It might save you time on reviews to ask them to post a QA link first